templates/azure/support.html

```diff
@@ -0,0 +1,145 @@
+{% extends "azure/base_azure.html" %}
+
+{% block title %}Ubuntu Pro with Support{% endblock %}
+{% block meta_description %}Ubuntu Pro for Azure, the Ubuntu image optimised for production and professional use on public cloud. Including broad security coverage, live kernel patching, certified components with hardening profiles, and backed by a 10-years maintenance commitment by Canonical. Get it on the Azure marketplace.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1XjmCkpSjHMLdc6Hdn4L8ml9sQL2wZWfwZg_h94dcgMU/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--square-darksuru is-dark">
+  <div class="row">
+    <div class="col-6">
+      <h1>Ubuntu Pro with Support</h1>
+      <p>Ubuntu Pro with Support provides a supported, secure foundation for modern applications by adding 24x7 technical support for your Ubuntu Pro workloads on Azure, metered by the hour as a Private Offer.</p>
+      <p><a href="/contact-us" class="p-button--positive js-invoke-modal">Contact us</a></p>
+    </div>
+    <div class="col-6 u-align--center u-vertically-center u-hide--medium u-hide--small">
+      {{ image (
```

Comment:
This image looks slightly larger on the design, but I'm not sure if that's just my impression? Ignore this if so!